contributions/api/controllers/contributions.py

```diff
@@ -200,6 +200,11 @@ def post(token_info):
         msg_json = jsonutils.create_log_json("Contribution", "POST", {"id": str(contribution_id)})
         logging.info("Contribution POST " + json.dumps(msg_json))
 
+        # send new contribution email to add reviewers
+        list_reviewers = mongoutils.list_reviewers()
+        for reviewer in list_reviewers:
+            send_email_new_contribution(reviewer['githubUsername'])
+
```

Comment:
is it better to just send a single email to all reviewers at once, instead of sending emails to each individual reviewers. So maybe modify the send_mail_new_contributions to accept just a single username but the list of the usernames, so it can send only one email to all the members where they are just one person or multiple people

Reply:
The send_email_new_contribution() method will be modified to notify only the reviewer who has been assigned to that specific contribution. Since the assignment of a reviewer to a contribution functionality is still in development, it might be better to have send_email_new_contribution() to take in one username and send email notification to that particular username